plugins/drivers/utils.go

```diff
@@ -441,6 +441,11 @@ func resourceUsageToProto(ru *ResourceUsage) *proto.TaskResourceUsage {
 			cpu.MeasuredFields = append(cpu.MeasuredFields, proto.CPUUsage_THROTTLED_TIME)
 		case "Percent":
 			cpu.Percent = ru.CpuStats.Percent
+
+			// total ticks is derived from percent, and should always be set together
+			// even if "Total Ticks" is not marked as an explicit measured field
```

Comment:
FWIW, I tried adding Total Ticks as an explicit measured field, and got the following alloc status output:

```
CPU Stats
Percent  System Mode  Total Ticks  User Mode
101.12%  0.00%        101.12%
```

So I opted to make it implicit and preserve 0.8 behavior as much as possible.